components/stream-provider/azure-pipelines.yml

```diff
@@ -0,0 +1,23 @@
+trigger:
+  branches:
+    include:
+    - repo-restructure
+    - master
+  paths:
+    exclude:
+    - stream-consumer
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+steps:
+- script: docker build . -f components/stream-provider/Dockerfile -t bengreenier/browserd/stream-provider:$(Build.BuildId) -t bengreenier/browserd/stream-provider:latest
```

Comment:
we should not change the tag here (ie: bengreenier/browserd => bengreenier/browserd/stream-provider) as it will wreck the https://hub.docker.com/r/bengreenier/browserd/tags setup